### PR TITLE
https over http

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: index.html.haml [hunk 1]
```diff
@@ -80,10 +80,10 @@ layout: base
           Drools Fusion (complex event processing features)
       %li
         %img(src = "headerFooter/flow_icon_32x.png")
-          %a(href="http://www.jboss.org/jbpm") jBPM (process/workflow integration for rule orchestration/flow)
+          %a(href="https://www.jboss.org/jbpm") jBPM (process/workflow integration for rule orchestration/flow)
```
Comment (psiroky) [inline index.html.haml]: We should use https://www.jbpm.org here. @mbiarnes could you please fix that?

Comment (mbiarnes) [inline index.html.haml]: @psiroky isn't this what I did?

Comment (psiroky) [inline index.html.haml]: jboss.org/jbpm vs jbpm.org
It's not strictly related to the changes you did, but we should fix it anyway.

**Comment by ge0ffrey** [comment]
+1

**Pull request merged**